server.ts

```diff
@@ -1,10 +1,9 @@
 import express from 'express'
 import smartApp from './smartapp'
-import {Request, ParamsDictionary, Response} from "express-serve-static-core";
 
 
 const server = express();
-const PORT = 3001;
+const PORT = process.env.PORT || 3001;
```

Comment:
Since this is TypeScript, you could use the more specific/intentional/modern `??` operator here.

Reply:
Great suggestion. [Added here](https://github.com/SmartThingsCommunity/smartapp-example-open-close-nodets/compare/e6d495233401f89503a8d2f5970d11a5b89b9d9f..b8b5efe003e76aaa29950da582c55e0806dfb18b).